site/src/pages/CreateWorkspacePage/CreateWorkspacePageViewExperimental.tsx

```diff
@@ -404,11 +404,6 @@ export const CreateWorkspacePageViewExperimental: FC<
 							</Tooltip>
 						</TooltipProvider>
 					</span>
-					<FeatureStageBadge
```

Comment:
Should we be removing the badge for dynamic params as well? AFAIK that's still in beta for a bit longer.